Sources/App/Interactors/CommentErrorInteractor.swift

```diff
@@ -0,0 +1,34 @@
+import Vapor
+
+public struct CommentErrorInteractor {
+    func execute(on req: Request, build: CircleCIBuild, repo: String, message: String) -> Future<Response> {
+        guard let github = try? req.make(GithubService.self) else {
+            return req.future(
+                error: ServiceError(identifier: "Github", reason: "GithubService could not be created")
+            )
+        }
+        
+        guard build.pullRequests.count > 0 else {
+            return req.future(error: Abort(HTTPStatus.ok))
+        }
+        let pullRequest = build.pullRequests[0]
+        
+        guard
+            let issueNumberString = pullRequest.url.absoluteString.split(separator: "/").last,
+            let issueNumber = Int(issueNumberString)
+        else {
+            return req.future(error: Abort(.notFound))
```

Comment:
I don't think GH likes it when you send back 404s. Might even trigger them to re-send the webhook, though I'm not sure.

Maybe change this to a 200, because GH expects only 200s, and that way you can also merge the above guard statement and this one into 1 big one:
```swift
guard
    let pullRequest = build.pullRequests.first,
    let issueNumberString = pullRequest.url.absoluteString.split(separator: "/").last,
    let issueNumber = Int(issueNumberString)
else {
    return req.future(error: Abort(.ok))
}

Reply:
Ah yeah. We should go through and add some actual logging next. I was using the failed responses to figure out when the bot failed, but returning 200 and logging the error would be more helpful.